CHANGELOG.md

```diff
@@ -36,6 +37,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 -   Set minimum required node version to 18.18.0 (required by `@typescript-eslint` v7)
 -   Update documentation on modules components
 -   Update required dependencies, move `react` and `react-dom` to peer dependencies
+-   Update `Tag` component primary variant styling
```

Comment:
This should be under the `unreleased` header.. We should think about using a tool like [changesets](https://github.com/changesets/changesets) to avoid all these conflicts/issues with the changelog 😅

Reply:
Damn. Yeah, let me look at that because it's rarely the case that we don't run into a changelog issue, and if you're not careful, stuff will just sneak in. Good catch.